go/vt/tabletserver/memcache_stats.go

```diff
@@ -240,11 +255,11 @@ func (s *MemcacheStats) updateSlabsStats() {
 			return
 		}
 		if slabsSingleMetrics[sKey] {
-			m, ok := s.slabs[sKey]
+			m, ok := memstats.slabs[sKey]
 			if !ok {
-				log.Errorf("Unknown memcache slabs stats %v: %v", sKey, ival)
-				internalErrors.Add("MemcacheStats", 1)
-				return
+				panic(fmt.Sprintf("impossible to reach here, slabs should always "+
```

Comment:
I just thought it is impossible to have a key in slabsSingleMetrics but not in MemcacheStats.slabs (see publishSlabsStats). But if you think panic is too aggressive, I could use varz instead.